### ISSUE TITLE
logging needs to log to database because of multithreading

### ISSUE THREAD

**Issue by urapadmin** [urapadmin/kiosk#272]
see #11

**Comment by urapadmin** [comment]
Is that really the case? Why can't I give every thread its own log file and stick with text files? 
The text files have at least one advantage: Accessibility. If the database fails, no logging would happen.

So what about just using a log per process/thread?
